src/main/resources/templates/db_scripts/schema.json

```diff
@@ -469,6 +451,47 @@
           "tOps": "ADD"
         }
       ]
+    },
+    {
+      "tableName": "invoice_transaction_summaries",
+      "fromModuleVersion": "mod-finance-storage-4.0.0"
+    },
+    {
+      "tableName": "temporary_invoice_transactions",
+      "fromModuleVersion": "mod-finance-storage-4.0.0",
+      "uniqueIndex": [
+        {
```

Comment:
Could you add this unique index to transaction table with where transactionType=Creadit or transactionType=Payment?
I think it'll help protect the transaction from being processed again.

Reply:
done